assets/src/components/v2/persistent_wrapper.tsx

```diff
@@ -37,7 +37,7 @@ const PersistentWrapper: ComponentType<Props> = ({
       {...visibleData}
       onFinish={handleFinished}
       key={renderKey}
-      lastUpdate={lastFetch}
+      lastUpdate={lastFetch} // make this optional - either determined by lastFetch Context or by other interval
```

Comment:
This was something I originally talked about with Cora, but there isn't really an equivalent for `lastUpdate` to trigger the hook in wrappedComponent. Not sure if this wrapper makes as much sense anymore to use for the elevator screen components